repos/system_upgrade/el7toel8/actors/verifydialogs/libraries/library.py

```diff
@@ -0,0 +1,20 @@
+from leapp.libraries.stdlib import api
+from leapp.models import DialogModel
+from leapp import reporting
+
+
+def check_dialogs(inhibit_if_no_userchoice=True):
+    results = list(api.consume(DialogModel))
+    for dialog in results:
+        sections = dialog.answerfile_sections.split(',')
+        dialog_resources = [reporting.RelatedResource('dialog', s) for s in sections]
+        dialogs_remediation = ('Please register user choices with leapp answer cli command or by manually editing '
+                               'the answerfile.')
```

Comment:
I know the "answerfile" location is quite visible at the end of the log but I would probably mention it in the report remediation hint too. 

_Please register user choices with leapp answer cli command or by manually editing the answerfile in ..._

Reply:
Double checked and for now there is no nice way to retrieve answerfile location from the actor. I don't want to hardcode a magic constant (which is likely to change in the future as per discussion with @pirat89 I agree that /var/log/leapp/answerfile is not a proper way to have an answerfile anyway) and as long as answerfile location is fairly visible I'd leave that change for the next patch.